mappings/net/minecraft/client/world/FlatWorldGeneratorPresets.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/unmapped/C_brlntonw net/minecraft/client/world/FlatWorldGeneratorPresets
+	METHOD m_qyjelmkr register (Ljava/lang/String;)Lnet/minecraft/unmapped/C_xhhleach;
+		ARG 0 id
+	METHOD m_sybrxjpq getHolder ()Lnet/minecraft/unmapped/C_cjzoxshv;
+	CLASS C_gyvjwgmv Bootstrap
+		FIELD f_bwhvkwau biomes Lnet/minecraft/unmapped/C_tqxyjqsk;
+		FIELD f_ogvgseei flatWorldGeneratorPresetsWorldgen Lnet/minecraft/unmapped/C_tqxyjqsk;
```

Comment:
The worldgen prefix looks out of place

Reply:
this is a registry, so the field name just copies the registry name